amundsen_application/static/js/components/TableDetail/DataPreviewButton/index.tsx

```diff
@@ -22,15 +22,11 @@ export interface StateFromProps {
   status: LoadingStatus;
 }
 
-export interface DispatchFromProps {
-  getPreviewData: (queryParams: PreviewQueryParams) => void;
-}
-
 export interface ComponentProps {
   queryParams: PreviewQueryParams;
```

Comment:
Now that the `DataPreviewButton` no longer makes the request to fetch data, `queryParams` is only used to generate the modal title. It might make sense to pass in `"title"` or `"modalTitle"` instead.

Reply:
Agreed, will pass a `modalTitle`.